ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml

```diff
@@ -808,6 +808,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-usertags
+- as: aws-ipi-kerneltype-f28-64k-pages
+  cron: 14 4 13 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @network-ovnkubernetes and not @fips'
+      MCO_CONF_INSTALL_KERNEL_TYPE: 64k-pages
+      OCP_ARCH: arm64
+      TAG_VERSION: '@4.15'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-kerneltype
```

Comment:
```yaml
- as: aws-ipi-kerneltype-f28-64k-pages
  cron: 14 4 13 * *
  steps:
    cluster_profile: aws-qe
    dependencies:
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
    env:
      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
      ADDITIONAL_WORKER_VM_TYPE: m6i.xlarge
      ADDITIONAL_WORKERS: '1'
      BASE_DOMAIN: qe.devcluster.openshift.com
      COMPUTE_NODE_TYPE: m6g.xlarge
      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
      E2E_RUN_TAGS: '@heterogeneous and @aws-ipi and @network-ovnkubernetes and not @fips'
      MCO_CONF_INSTALL_KERNEL_TYPE: 64k-pages
      OCP_ARCH: arm64
      TAG_VERSION: '@4.15'
    test:
     - ref: ipi-install-heterogeneous
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-aws-ipi-kerneltype
```

Can you change the test config with this one, and add in the `__multi` test config yaml file? We are trying to reduce the number of test configs based on the single-arch payload.

Other than that, lgtm.

Reply:
Hello! I created this configuration, but the new amd node cannot join. MCO_CONF_INSTALL_KERNEL_TYPE configures the kerneltype  using day-1. When the new node is created in the "heterogeneous" step we try to apply the 64k-pages kernel to the new node as well and it can't join. It seems that we can only use day-1 kerneltype in single arch clusters.

I have created another configuration too, using day-2 operations to create a new custom pool and  apply the 64k-pages kernel to that new pool. Could you please review it?

/cc @rioliu-rh  could you have a look at it too, if it is not what you expected, please tell me and I'll change it.

Reply:
As for the TC in the suite, to support multi-arch compute nodes, regardless of the day1/day2 'concept', you must have a dedicated machine config pool:
- the MCP bound to the 64k pagesize kernel can only host arm64 machines (the same is for rt-kernel: the MCP bound to the rt-kernel MC can only host amd64 machines)
- Other MCPs can host both arm64 and amd64 nodes.

Reply:
I didn't dig into the new day1 steps, but if you want it, the same steps can handle both day1 and day2 without duplicating code.

- Always drop the created files in the SHARED_DIR folder with their name being `manifest_(mcp|mc)_xxx.yaml`. 
- At the end of the step, if some after-installation file exists in the `$SHARED_DIR` (e.g., ` ${SHARED_DIR}/success`, see [this](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L679-L687)), you also run `oc create/apply -f $SHARED_DIR/manifest_mcp|mc_xxx.yaml`
- if the step runs before ipi-install,  [this snippet](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L595-L597) will take care of including the manifests at "day1" and no `oc apply` was run before.

Reply:
first thing I wanna clarify, the op of kernel type for day1 and day2 are different.
- day1, the logic is very simple, just put the manifest file to shared dir
- day2. we need to do more ops. create mc, apply it, wait mcp update to complete.

2nd thing, our code/design need to be user friendly. easy understand, the common naming pattern is `conf` dir is for day1 steps. it is confused to me, why we design a step in day1 dir to support day2 op

@sergiordlr current impl LGTM. thx